py/desitarget/data/targetmask.yaml

```diff
@@ -24,14 +24,14 @@ desi_mask:
     - [STD_WD,      34, "White Dwarf stars", {obsconditions: DARK|GRAY}]
     - [STD_BRIGHT,  35, "F-type standard for BRIGHT conditions",
         {obsconditions: BRIGHT}]
+    - [BADSKY,      36, "Blank sky locations that are imperfect but still useable",
+        {obsconditions: DARK|GRAY|BRIGHT|POOR|TWILIGHT12|TWILIGHT18}]
```

Comment:
I moved this because I started to think of the meaning of the bit more as a (bad) sky fiber location rather than as a masked location. I believe that we will need to have other BADSKY locations that are unrelated to masks, according to the argument laid out in this TechNote:

https://desi.lbl.gov/DocDB/cgi-bin/private/RetrieveFile?docid=2345;filename=blank-skies.pdf;version=3

I'm certainly happy to return this to bit 53, if you think it makes more logical sense, though.